Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.WebContent.sb.in

```diff
@@ -503,39 +502,37 @@
 
 (with-filter (system-attribute apple-internal)
     (allow network-outbound
-        (literal "/private/var/run/syslog"))
-)
+        (literal "/private/var/run/syslog")))
 
 (define (notifyd-message-numbers) (message-number 1002 1011 1012 1016 1017 1018 1021 1025 1026 1028 1029 1030 1031 1032))
-
-(allow mach-lookup (global-name "com.apple.system.notification_center")
-    (apply-message-filter
-        (deny mach-message-send)
-        (deny mach-message-send (with no-report) (message-number 1023))
-        (allow mach-message-send (notifyd-message-numbers))))
-
-(allow ipc-posix-shm-read*
-    (ipc-posix-name "apple.shm.notification_center"))
-
-#if ENABLE(NOTIFY_BLOCKING)
-(with-filter (state-flag "EnableExperimentalSandbox")
+(define (allow-notifyd-messages)
     (allow mach-lookup (global-name "com.apple.system.notification_center")
         (apply-message-filter
             (deny mach-message-send)
             (deny mach-message-send (with no-report) (message-number 1023))
-            (deny mach-message-send (with telemetry-backtrace) (notifyd-message-numbers)))))
+            (allow mach-message-send (notifyd-message-numbers)))))
+
+#if ENABLE(NOTIFY_BLOCKING)
+(with-filter (notify-blocking)
+    (deny mach-lookup (global-name "com.apple.system.notification_center")))
```

Comment:
Based on code, it seems if we don't explicitly `allow mach-lookup (global-name "com.apple.system.notification_center"))`, the default behavior is deny, right?
Then this should not be necessary?